src/util/pg.py

```diff
@@ -55,6 +55,14 @@ class PGRegexp(str):
     """
 
 
+class SQLStr(str):
+    """
+    Wrapper for semantic meaning of: this string is a valid snippet SQL.
```

Comment:
Nitpick: the meaning is for when used as parameter.
```suggestion
    Wrapper for semantic meaning of parameters: this string is a valid SQL snippet.
```